node_modules/oae-authentication/emailTemplates/reset.html.jst

```diff
@@ -0,0 +1,190 @@
+<%
+    /**
+     * Render the email header
+     */
+    var renderHeader = function() {
+        var institutionalLogoURL = shared.ensureAbsoluteLink(util.ui.getHashedPath(skinVariables['institutional-logo-url'].replace(/\'/g, '')), baseUrl);
```

Comment:
Yes. It will load `OAE` logo by default. Let me know if we need to do further checks on this.